sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobStorageResourceContainer.cs

```diff
@@ -14,54 +14,59 @@
 namespace Azure.Storage.DataMovement.Blobs
 {
     /// <summary>
-    /// Blob Container Resource (no directories because of the flat namespace)
+    /// The Storage Resource class for the Blob Client. Directories are not supported
+    /// as blobs only support a flat namespace structure. See <see cref="BlobDirectoryStorageResourceContainer"/>
+    /// for directory support.
     /// </summary>
     public class BlobStorageResourceContainer : StorageResourceContainer
     {
         private BlobContainerClient _blobContainerClient;
         private BlobStorageResourceContainerOptions _options;
 
         /// <summary>
-        /// Constructor
+        /// The constructor to create an instance of the BlobStorageResourceContainer.
         /// </summary>
-        /// <param name="blobContainerClient"></param>
-        /// <param name="options"></param>
+        /// <param name="blobContainerClient">
+        /// The blob client which represents the storage container
+        /// to perform the transfer source or destination.
+        /// </param>
+        /// <param name="options">Options for the storage resource. See <see cref="BlobStorageResourceContainerOptions"/>.</param>
         public BlobStorageResourceContainer(BlobContainerClient blobContainerClient, BlobStorageResourceContainerOptions options = default)
         {
             _blobContainerClient = blobContainerClient;
             _options = options;
         }
 
         /// <summary>
-        /// Can produce uri
+        /// Defines whether the storage resource type can produce a SAS URL.
         /// </summary>
         public override ProduceUriType CanProduceUri => ProduceUriType.ProducesUri;
 
         /// <summary>
-        /// Returns the path
+        /// Gets the path of the storage resource.
         /// </summary>
         public override string Path => _blobContainerClient.Name;
 
         /// <summary>
-        /// Obtains the Uri of the blob directory resource, which means we can list
+        /// Gets the URL of the storage resource. If the URL can be obtained, this storage resource can be listed.
         /// </summary>
         public override Uri Uri => _blobContainerClient.Uri;
 
         /// <summary>
-        /// Creates new blob client based on the parent container client
+        /// Retrieves a single blob resource based on this respective resource.
         /// </summary>
-        /// <param name="path"></param>
+        /// <param name="path">The path to the storage resource.</param>
```

Comment:
Want to check if this should be changed to `encodedPath` or if that is inaccurate? Not that there is anything wrong with `path`, but rather that this is the only place where the parameter is named `path`.